app/packages/core/src/components/ColorModal/LabelTag.tsx

```diff
@@ -90,52 +105,34 @@ const LabelTag: React.FC = () => {
     }
   };
 
-  // initialize field settings
-  useEffect(() => {
-    // check setting to see if custom setting exists
-    const copy = cloneDeep(labelTags);
-    if (!copy || _.isEmpty(copy)) {
-      const defaultSetting = {
-        fieldColor: color,
-        valueColors: [],
-      } as fos.CustomizeColor;
-      setColorScheme(false, { colorPool, fields, labelTags: defaultSetting });
-      setState({
-        useLabelColors: Boolean(
-          labelTags?.valueColors && labelTags.valueColors.length > 0
-        ),
-        useFieldColor: Boolean(labelTags?.fieldColor),
-      });
-    }
-  }, [labelTags]);
-
   fos.useOutsideClick(wrapperRef, () => {
     setShowFieldPicker(false);
   });
 
+  useEffect(() => {
+    setInput(labelTags?.fieldColor);
```

Comment:
why are are using conditional property access for `labelTags?.X`. It seems to be defined at all times no? useEffect above should set it